app/controllers/annotation_categories_controller.rb

```diff
@@ -204,6 +204,11 @@ def upload
     redirect_to assignment_annotation_categories_path(assignment_id: @assignment.id)
   end
 
+  def annotation_text_stats
+    annotation_stats = AnnotationText.find(params[:annotation_text_id]).stats
+    render json: annotation_stats
```

Comment:
You don't need a temporary variable here, you can do `render json: AnnotationText....`